browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -210,6 +215,30 @@ ShowTranslateBubbleResult BraveBrowserView::ShowTranslateBubble(
   return ShowTranslateBubbleResult::BROWSER_WINDOW_NOT_VALID;
 }
 
+WalletButton* BraveBrowserView::GetWalletButton() {
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+  return static_cast<BraveToolbarView*>(toolbar())->wallet_button();
+#else
+  return nullptr;
+#endif
+}
+
+void BraveBrowserView::CreateWalletBubble() {
+  // Do not spawn the bubble if using the WebUITabStrip.
+#if BUILDFLAG(ENABLE_WEBUI_TAB_STRIP)
```

Comment:
so no bubble on windows or linux? What happens there?

Reply:
Addressed in 3776c2612b71b565d46683a4adc3bb98430355e7